### PR TITLE
Rename app name

### LINKED ISSUE #1045: Wipe out the word Mesa from this interface
Mesa is an App managed by dxDAO, this is a general UI for Gnosis Protocol. We should remove any reference to Mesa to avoid confusion.

Also, we should bring back the `GP` that we had. I think it's still in the project.

### PR DESCRIPTION
**Pull request by anxolin**
Closes #1045 

* Rename the app to Gnosis Protocol Web
* Removes the old log. Now it uses `df` logo because I don't find the one I would swear we had at one point with `GP`. @biocom do you have a the logo with `GP`?
* Adapt files like readme to say Gnosis Protocol Web
* For components like the footer or the about, it uses a new global var for the name, so it's automatically loaded from the config
* For the FAQ is just renamed for simplicity. Probably forks of this projects would create their own FAQ

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1048--dexreact.review.gnosisdev.com

File: config-default.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #   - For redefining components, or for more information, follow the instructions in ./README.md
 
 # TODO: Rename Dapp into Gnosis Protocol DApp
```
Comment (alfetopito) [inline config-default.yaml:10]: Remove TODO:

```suggestion
```

File: webpack.config.babel.js [hunk 2]
```diff
@@ -177,6 +177,7 @@ module.exports = ({ stats = false } = {}) => ({
       DEX_JS_VERSION: JSON.stringify(require('@gnosis.pm/dex-js/package.json').version),
       CONTRACT_VERSION: JSON.stringify(require('@gnosis.pm/dex-contracts/package.json').version),
       CONFIG: JSON.stringify(config),
+      APP_NAME: JSON.stringify(appName),
```
Comment (alfetopito) [inline webpack.config.babel.js:180]: Why not import that from `CONFIG` object instead?

Comment (anxolin) [reply webpack.config.babel.js:180]: Right! no reason, that's me being silly

File: src/components/TradeWidget/TokenRow.tsx [hunk 3]
```diff
@@ -139,8 +139,8 @@ interface Props {
 
 const BalanceTooltip = (
   <HelpTooltipContainer>
-    This balance reflects the amount deposited in the Exchange Wallet on Mesa. Only orders with a balance will be
-    considered for matching.
+    This balance reflects the amount deposited in the Exchange Wallet on Gnosis Protocol. Only orders with a balance
```
Comment (W3stside) [inline src/components/TradeWidget/TokenRow.tsx:142]:
```suggestion
    This balance reflects the amount deposited in the Exchange Wallet on Gnosis Protocol Web. Only orders with a balance
```

File: src/pages/About.tsx [hunk 4]
```diff
@@ -40,9 +40,9 @@ const About: React.FC = () => {
   })
   return (
     <AboutWrapper>
-      <h1>About Mesa</h1>
+      <h1>About {APP_NAME}</h1>
       <p>
-        Mesa it&apos;s an Open Source interface for{' '}
+        <strong>{APP_NAME}</strong> it&apos;s an Open Source interface for{' '}
```
Comment (W3stside) [inline src/pages/About.tsx:45]:
```suggestion
        <strong>{APP_NAME}</strong> is an Open Source interface for{' '}
```

File: src/components/TradeWidget/TokenRow.tsx (hunk 3 above)
Comment (anxolin) [reply src/components/TradeWidget/TokenRow.tsx:142]: @W3stside I did this one on purpose. The balance is in the contract, so I said Gnosis Protocol. I didn't want to refer to the UI. If I wanted to do that, also I wouldn't have "hardcode" it --> I could use the app name config

**Pull request merged**